modules/cnf-programming-the-sriov-fec-operator-acc100.adoc

```diff
@@ -0,0 +1,345 @@
+
++
+[source,terminal]
+----
+$ oc get sriovfecnodeconfig node1 -o yaml
+----
++
+.Example output
+[source,yaml]
+----
+status:
+    conditions:
+    - lastTransitionTime: "2021-03-19T17:19:37Z"
+      message: Configured successfully
+      observedGeneration: 1
+      reason: ConfigurationSucceeded
+      status: "True"
+      type: Configured
+    inventory:
+       sriovAccelerators:
+       - deviceID: 0d5c
+         driver: ""
+         maxVirtualFunctions: 16
+         pciAddress: 0000:af:00.0 <1>
+         vendorID: "8086"
+         virtualFunctions: [] <2>
+----
+<1> This field indicates the PCI address of the card.
+<2> This field shows that the virtual functions are empty.
+
+.  Configure the FEC device with the desired setting.
+
+.. Create the following custom resource (CR) and save the YAML in the `sriovfec_acc100cr.yaml` file.
++
+[NOTE]
+====
+This example configures the ACC100 8/8 queue groups for 5G, 4 queue groups for Uplink, and another 4 queue groups for Downlink.
+====
++
+[source,yaml]
+----
+apiVersion: sriovfec.intel.com/v1
+kind: SriovFecClusterConfig
+metadata:
+  name: config <1>
+spec:
+  nodes:
+   - nodeName: node1 <2>
+     physicalFunctions:
+       - pciAddress: 0000:af:00.0 <3>
+         pfDriver: "pci-pf-stub"
+         vfDriver: "vfio-pci"
+         vfAmount: 16 <4>
+         bbDevConfig:
+           acc100:
+             # Programming mode: 0 = VF Programming, 1 = PF Programming
+             pfMode: false
+             numVfBundles: 16
+             maxQueueSize: 1024
+             uplink4G:
+               numQueueGroups: 0
+               numAqsPerGroups: 16
+               aqDepthLog2: 4
+             downlink4G:
+              numQueueGroups: 0
+              numAqsPerGroups: 16
+              aqDepthLog2: 4
+             uplink5G:
+              numQueueGroups: 4
+              numAqsPerGroups: 16
+              aqDepthLog2: 4
+             downlink5G:
+              numQueueGroups: 4
+              numAqsPerGroups: 16
+              aqDepthLog2: 4
+----
+<1> Specify a name for the CR object. The only name that can be specified is `config`.
+<2> Specify the node name.
+<3> Specify the PCI address of the card on which the SR-IOV-FEC Operator will be installed.
+<3> Specify the number of virtual functions to create. For the Intel® vRAN Dedicated Accelerator ACC100, create all 16 VFs.
++
+[NOTE]
+====
+The card is configured to provide up to 8 queue groups with up to 16 queues per group. The queue groups can be divided between groups allocated to 5G and 4G and Uplink and Downlink.
+The Intel® vRAN Dedicated Accelerator ACC100 can be configured for:
+
+* 4G or 5G only
+* 4G and 5G at the same time
+
+Each configured VF has access to all the queues. Each of the queue groups have a distinct priority level. The request for a given queue group is made from the application level that is, the vRAN application leveraging the FEC device.
+====
+
+.. Apply the CR.
```

Comment:
End the instructional text with a colon.

Ideally, you would also indicate what is configured by applying the YAML.